scripts/aws/iam/create-user_deploy.sh

```diff
@@ -1 +1,3 @@
-aws iam create-user --user-name deploy
+aws iam create-user --user-name deploy \
+  | jq ".AccessKey.SecretAccessKey" \
```

Comment:
You're doing nothing with these keys right now so they are not saved anywhere and would need to be redone through the aws console. Small negative.